test/unit-tests/ima_test.js

```diff
@@ -516,7 +521,52 @@ describe('ad_manager_ima', function()
       }
       ima.loadMetadata(metadata, {}, {});
       expect(ima.showAdControls).to.be(false);
+      expect(ima.autoHideAdControls).to.be(true);
+    });
+
+    it('Autohide ad controls should be enabled if we are showing the controls over the ads.', function()
+    {
+      amc.pageSettings.playerControlsOverAds = true;
+      var metadata = {
+        showAdControls: true
+      }
+      ima.loadMetadata(metadata, {}, {});
+      expect(ima.showAdControls).to.be(true);
+      expect(ima.autoHideAdControls).to.be(true);
     });
+
+    it('Play ad: Requests the AMC to hide the player UI', function()
+    {
+      var notified = false;
+      amc.hidePlayerUi = function(showAdControls, showAdMarquee, autoHideAdControls) {
+        expect(showAdControls).to.be(false);
+        expect(autoHideAdControls).to.be(true);
+        notified = true;
+      };
+      initAndPlay(true, vci);
+      ima.playAd(amc.timeline[0]);
+      var am = google.ima.adManagerInstance;
+      am.publishEvent(google.ima.AdEvent.Type.STARTED);
+      expect(notified).to.be(true);
+    });
+
+    it('Play ad: Requests the AMC to hide the player UI', function()
```

Comment:
Can we differentiate this description from the previous unit test? They currently have the same description

Reply:
Will do. Copy-pasting is bad :(